proxstar/vnc.py

```diff
@@ -13,14 +14,18 @@
 def stop_websockify():
     result = subprocess.run(['pgrep', 'websockify'], stdout=subprocess.PIPE, check=False)
     if result.stdout:
-        pid = result.stdout.strip()
-        subprocess.run(['kill', pid], stdout=subprocess.PIPE, check=False)
-        time.sleep(3)
-        if subprocess.run(['pgrep', 'websockify'], stdout=subprocess.PIPE, check=False).stdout:
-            time.sleep(10)
+        pids = result.stdout.splitlines()
+        for pid in pids:
+            subprocess.run(['kill', pid], stdout=subprocess.PIPE, check=False)
+            # FIXME (willnilges): Willard is lazy.
+            time.sleep(1)
```

Comment:
Why is there a sleep? Do you use stdout that you need to pipe it?

Reply:
IDK, ask Jordan.

Reply:
I think the real reason is that it sends the `kill` command and then waits to see if that actually worked. If it didn't, then it `kill -9`'s the process.